code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -28,16 +28,16 @@
 #define STAMINALOSS_REGEN_COEFF 50
 
 //Thresholds for detrimental effects from stamina
-#define STAMINA_THRESHOLD_WEAK 60
+#define STAMINA_THRESHOLD_WEAK 25
```

Comment:
This is more than 50% less than what it was. We should be starting small. Not the other way round.